principia_make.sh

```diff
@@ -1,3 +1,8 @@
+# Define AGENT_OS if not present.
```

Comment:
`principia_make.sh` (which exists in the dependency repositories as well) is not meant to be run directly. It is used by the Azure pipelines that run the continuous integration, and here it mostly contains workarounds to please Azure (the term « agent » and the variable `AGENT_OS` both come from Azure).

FWIW the calling code is as follows.
```bash
if [ -f ./principia_variable_overrides.sh ]; then
  source ./principia_variable_overrides.sh
fi

if [ "${AGENT_OS?}" == "Darwin" ]; then
  C_FLAGS="${PRINCIPIA_C_FLAGS?} -mmacosx-version-min=${PRINCIPIA_MACOS_VERSION_MIN?} -arch x86_64"
  CXX_FLAGS="${PRINCIPIA_CXX_FLAGS?} ${PRINCIPIA_MACOS_CXX_FLAGS?}"
elif [ "${AGENT_OS?}" == "Linux" ]; then
  C_FLAGS="${PRINCIPIA_C_FLAGS?} -m64"
  CXX_FLAGS="${PRINCIPIA_CXX_FLAGS?}"
else
  C_FLAGS="${PRINCIPIA_C_FLAGS?}"
  CXX_FLAGS="${PRINCIPIA_CXX_FLAGS?}"
fi
LD_FLAGS="${C_FLAGS?} ${PRINCIPIA_LD_FLAGS?}"
CXX_FLAGS="${CXX_FLAGS?} ${LD_FLAGS?}"

# Stop on first failure.
set -e
source ./principia_make.sh
```

You probably want to run `make` directly when building Principia yourself.

Please revert the changes to this file.